lstchain/calib/camera/r0.py

```diff
@@ -84,6 +86,17 @@ class LSTR0Corrections(CameraR0Calibrator):
                             allow_none=True,
                             help='Path to the LST pedestal binary file'
                             ).tag(config=True)
+    calibration_path = Unicode(
+        '',
```

Comment:
If you allow_none, the default should be `None` as well.

Reply:
It makes sense, but that is not what is done elsewhere in the code.
Should I change that as well ?